_example/api-server/server.go

```diff
@@ -9,6 +9,11 @@ import (
 	rent "github.com/neighborhood999/fiveN1-rent-scraper"
 )
 
+type Response struct {
```

Comment:
I don't think `Response` is a good struct name as it is too general, which will be difficult and confusing for further extension.